packages/react-instantsearch-core/src/connectors/__tests__/connectHierarchicalMenu.js

```diff
@@ -174,6 +174,158 @@ describe('connectHierarchicalMenu', () => {
+        props,
+        searchState
+      );
+      const results = new SearchResults(state, [
+        {
+          hits: [],
+          renderingContent: {
+            facetOrdering: {
+              values: {
+                lvl0: {
+                  order: ['wat'],
+                },
+                lvl1: {
+                  order: ['wat > wut'],
+                },
+              },
+            },
+          },
+          facets: {
+            lvl0: {
+              wat: 20,
+              oy: 10,
+            },
+            lvl1: {
+              'wat > wot': 15,
+              'wat > wut': 5,
+            },
+          },
+        },
+      ]);
+
+      const providedProps = connect.getProvidedProps(props, searchState, {
+        results,
+      });
+      expect(providedProps.items).toEqual([
+        {
+          label: 'wat',
+          value: undefined,
+          count: 20,
+          isRefined: true,
+          items: [
+            {
+              label: 'wut',
+              value: 'wat > wut',
+              count: 5,
+              isRefined: false,
+              items: null,
+            },
+            {
+              label: 'wot',
+              value: 'wat > wot',
+              count: 15,
+              isRefined: false,
+              items: null,
+            },
+          ],
+        },
+        {
+          label: 'oy',
+          value: 'oy',
+          count: 10,
+          isRefined: false,
+          items: null,
+        },
+      ]);
+    });
+
+    it('facetValues results does not use facetOrdering if disabled', () => {
+      const props = {
+        attributes: ['lvl0', 'lvl1'],
+        facetOrdering: false,
+        contextValue,
+      };
+      const searchState = { hierarchicalMenu: { lvl0: 'wat' } };
+      const state = connect.getSearchParameters(
+        new SearchParameters(),
+        props,
+        searchState
+      );
+      const results = new SearchResults(state, [
+        {
+          hits: [],
+          renderingContent: {
+            facetOrdering: {
+              values: {
+                lvl0: {
+                  order: ['wat'],
+                },
+                lvl1: {
+                  order: ['wat > wut'],
+                },
+              },
+            },
+          },
+          facets: {
+            lvl0: {
+              wat: 20,
+              oy: 10,
+            },
+            lvl1: {
+              'wat > wot': 15,
+              'wat > wut': 5,
+            },
+          },
+        },
+      ]);
+
+      const providedProps = connect.getProvidedProps(props, searchState, {
+        results,
+      });
+      expect(providedProps.items).toEqual([
+        {
+          label: 'oy',
+          value: 'oy',
+          count: 10,
+          isRefined: false,
+          items: null,
+        },
+        {
+          label: 'wat',
+          value: undefined,
+          count: 20,
+          isRefined: true,
+          items: [
```

Comment:
```suggestion
          items: [ // sorted by count:desc by default
```

WDYT? That context is missing in either comment or in the test name, so I had to think hard to remember it.

Reply:
it's actually alphabetical, but in this small set it's also count